### PR TITLE
Feature/proxy env vars

### PR DESCRIPTION
**Pull request by DRuggeri**
Very straight forward edit - adds ability to use HTTP proxy as is found in other CPIs like OpenStack and Azure.

### CODE REVIEW

**Comment by DRuggeri** [comment]
ICLA just got submitted... hopefully the process is not enough to block this PR.

File: releases/bosh-google-cpi/bosh-google-cpi-25.10.0.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
[the remark this answers is not in the document]
Comment (DRuggeri) [reply releases/bosh-google-cpi/bosh-google-cpi-25.10.0.yml]: Sure... will pull commits into my fork's develop branch and move PR over to it.

File: jobs/google_cpi/spec [hunk 2]
```diff
@@ -137,3 +137,10 @@ properties:
     description: "Registry Client Key (PEM format)"
   registry.client.tls.cacert:
     description: "Registry Client Root CA certificates (PEM format)"
+
+  env.http_proxy:
+    description: Http proxy to connect to cloud API's
+  env.https_proxy:
+    description: Https proxy to connect to cloud API's
```
[the remark this answers is not in the document]
Comment (DRuggeri) [reply jobs/google_cpi/spec]: I've corrected both. Sorry about that... these definitions came from the OpenStack CPI spec file.

**Comment by DRuggeri** [comment]
I struggled a bit with the cherry-picking into develop because of a self-inflicted wound, but I think we're good now

**Comment by johnsonj** [comment]
thank you @DRuggeri! I'll let this go through CI. Targeting a release tomorrow.

**Comment by DRuggeri** [comment]
Excellent - thanks!

**Comment by johnsonj** [comment]
Release may not make it today. I'm resolving issues in our CI pipeline with updates in [BATS](https://github.com/cloudfoundry/bosh-acceptance-tests/). Will ping when I've got a better idea.

**Comment by johnsonj** [comment]
Released as [v26.0.0](http://bosh.io/releases/github.com/cloudfoundry-incubator/bosh-google-cpi-release?version=26.0.0) - thank you!

**Pull request merged**